lib/helpers/focusManager.js

```diff
@@ -1,7 +1,7 @@
 import findTabbable from './tabbable';
 
+const focusLaterElement = [];
```

Comment:
Since this is an array now... perhaps it should be changed to `focusLaterElements`?